node/src/components/consensus/highway_core/state.rs

```diff
@@ -421,8 +425,13 @@ impl<C: Context> State<C> {
         let vote = self.opt_vote(hash)?.clone();
         let opt_block = self.opt_block(hash);
         let value = opt_block.map(|block| block.value.clone());
-        // TODO: After LNC we won't always need all known endorsements.
-        let endorsed = self.endorsements().collect();
+        // TODO
```

Comment:
We need to store the explicit endorsement dependencies in `Vote` now — I guess that's what that `TODO` refers to?

Reply:
This `TODO` is about including only required endorsements, and not all that are known.

Reply:
But there's no unique set of required endorsements computable from the panorama.
We need to store the endorsement claims in `Vote`, I believe.